scilpy/tractograms/tractogram_operations.py

```diff
@@ -24,15 +25,35 @@
 from scipy.ndimage import map_coordinates
 from scipy.spatial import cKDTree
 
+from scilpy.tractanalysis.bundle_operations import uniformize_bundle_sft
+from scilpy.tractanalysis.streamlines_metrics import compute_tract_counts_map
 from scilpy.tractograms.streamline_operations import smooth_line_gaussian, \
     resample_streamlines_step_size, parallel_transport_streamline, \
-    cut_invalid_streamlines
+    cut_invalid_streamlines, compress_sft, cut_invalid_streamlines, \
+    remove_overlapping_points_streamlines, remove_single_point_streamlines
+from scilpy.tractograms.streamline_and_mask_operations import \
+    cut_outside_of_mask_streamlines
+from scilpy.utils.spatial import generate_rotation_matrix
 
 MIN_NB_POINTS = 10
 KEY_INDEX = np.concatenate((range(5), range(-1, -6, -1)))
 
 
 def shuffle_streamlines(sft, rng_seed=None):
+    """
+    Shuffle the streamlines of a tractogram.
+
+    Parameters
+    ----------
+    sft: StatefulTractogram
+        The tractogram to shuffle (will slice the streamline, DPS and DPP).
```

Comment:
"to slice" is a python explanation. Can we avoid nerdy terms? Simply, will shuffle?